src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1443,12 +1445,14 @@ public void Socket_ReceiveFromDnsEndPoint_Throws()
         }
 
         [Fact]
+        [ActiveIssue(DummyOSXPacketInfoIssue, PlatformID.OSX)]
```

Comment:
These are all failing because of an issue we believe we can fix or because of something that'll remain unsupported?